### DISCUSSION TITLE
Code browser: Skip folders with one nested folders

### DISCUSSION THREAD

**Discussion by hfhbd** [community/community#55209] [Mobile]
### Select Topic Area

Product Feedback

### Body

The java code structure uses package names as folder names, so the package com.example is store in the folders `com/example`, which is stored in `src/main`. This results into many folders with one subfolder. At GitHub web, these package folders are skipped, so you always go from `main` directly to your Java code stored in `example`. 

On mobile, you have to navigate to each folder manually.

**Comment by aconsuegra** [comment]
Hi @hfhbd, thanks for reaching out with your feature request! I think it makes total sense, so I'm going to create an internal issue to track it. Please, keep the feedback coming!
